apps/test/unit/util/curriculumRecommenderTest.js

```diff
@@ -36,14 +37,33 @@ describe('testRecommender', () => {
   it('adds score to curricula with desired duration', () => {
     const recommendedCurricula = getTestRecommendations(
       DURATION_TEST_COURSES,
-      'week',
+      'month',
```

Comment:
I switched this to 'month' so that I could individually test the `hasDesiredDuration` function without the new `hasSlightlyLongerDuration` function adding points to the `monthDurationCourse` curriculum for being slightly longer than the `weekDurationCourse`